src/types/TodoWithoutUser.ts

```diff
@@ -0,0 +1,6 @@
+export interface TodoWithoutUser {
+  id: number,
```

Comment:
Consider renaming `TodoWithoutUser` to `Todo` since it is already clear that the user is not included in the interface. 🤔